index.html

```diff
@@ -173,5 +173,18 @@ <h5>Skills</h5>
             </li>
         </ul>
     </section>
+    <div class="form-container">
+        <form action="https://formspree.io/f/xyyovkvo" method="post">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <input type="text" id="Name" name="Name" placeholder="Your name.." minlength="3" maxlength="30">
+            <input type="email" id="email" name="email" placeholder="  Your e-mail..">
+            <textarea id="message" name="subject" placeholder="Write your message here"></textarea>
+            <input type="submit" value="Get in touch">
```

Comment:
-  [ ] please Required should be added to the form elements so that it won't submit unless all of them are present.